core/Updater.php

```diff
@@ -87,22 +116,13 @@ public static function getCurrentRecordedComponentVersion($name)
     }
 
     /**
-     * Returns the flag name to use in the option table to record current schema version
-     * @param string $name
-     * @return string
-     */
-    private static function getNameInOptionTable($name)
-    {
-        return 'version_' . $name;
-    }
-
-
-    /**
-     * This method ensures that Piwik Platform cannot be running when using a NEWER database
+     * This method ensures that Piwik Platform cannot be running when using a NEWER database.
+     *
+     * TODO: can I move this to FrontController & deprecate this?
```

Comment:
Sure, you can move it, and don't need to deprecate it, since it's not public API